extensions/amp-story/amp-story.md

```diff
@@ -221,6 +295,22 @@ Disables the [Live story](#Live-story) functionality.
 
 Used with the live-story attribute. Time interval (in milliseconds) between checks for new content. If no `data-poll-interval` is provided it with default to the 15000 millisecond minimum. A value under 15000 milliseconds is invalid.
 
+##### parallax-fx [optional]
```

Comment:
Could you please revert the formatting changes not related to this PR? We should keep them for a separate PR, as this one is already rather large